test/targets.cpp

```diff
@@ -50,4 +51,21 @@ TEST_CASE(targets)
     EXPECT(ts.size() >= 1);
 }
 
+TEST_CASE(concurrent_targets)
+{
+    std::vector<std::thread> threads;
+
+    for(auto i = 0u; i < 1000; i++)
```

Comment:
This should use a multiple of the hardware conccurency: `std::thread::hardware_concurrency() * 4` or something like that.

Reply:
done